app/backend/src/tests/test_threads.py

```diff
@@ -0,0 +1,125 @@
+import contextlib
+import string
+
+import grpc
+import pytest
+from google.protobuf import empty_pb2, wrappers_pb2
+from sqlalchemy.sql import and_
+
+from couchers import errors
+from couchers.db import session_scope
+from couchers.models import Thread
+from couchers.servicers.threads import Threads
+from couchers.utils import now
+from pb import threads_pb2, threads_pb2_grpc
+from tests.test_fixtures import api_session, db, fake_channel, generate_user, testconfig
+
+
+@pytest.fixture(autouse=True)
+def _(testconfig):
+    pass
+
+
+@contextlib.contextmanager
+def threads_session(token):
+    """
+    Create a communities API for testing, uses the token for auth
+    """
+    channel = fake_channel(token)
+    threads_pb2_grpc.add_ThreadsServicer_to_server(Threads(), channel)
+    yield threads_pb2_grpc.ThreadsStub(channel)
```

Comment:
Should this go into `text_fixtures` like all other `*_session`s? Sometimes tests from other parts still need/want to call the APIs from another session. These are also fairly similar, so it's easier to refactor (as you've done in this PR with `fake_channel`) if they're all in the one place.

Reply:
No, let's move it later on when it is used by other test files. Regarding refactoring, it is as easy to refactor no matter how many files you touch (same amount of lines to be changed)